src/summarycode/summarizer.py

```diff
@@ -161,6 +162,8 @@ def get_declared_holders(codebase, holders_tallies):
     A declared holder is a copyright holder present in the key files who has the
     highest amount of refrences throughout the codebase.
     """
+    warnings.simplefilter('ignore', UnicodeWarning)
```

Comment:
Did this work for you?

Can you reproduce this warning showing up and then after adding this bit of code verify that it went away?

I tried and for me it didn't went away.

```
main_repos/gh-cli-repos/scancode-toolkit  silence-pyicu-warning ✔                                                                                                                                         1d13h  ⍉
▶ scancode -clipeu --classify --summary --license-clarity-score --license-text ../../../write_access/scancode-extra/tests/formattedcode/data/yaml/with-package-and-license --yaml output.yaml
Setup plugins...
Collect file inventory...
Scan files for: info, packages, licenses, copyrights, emails, urls with 1 process(es)...
[####################] 8
/home/ayansinha/nexB/main_repos/gh-cli-repos/scancode-toolkit/venv/lib/python3.8/site-packages/fingerprints/cleanup.py:54: ICUWarning: Install 'pyicu' for better text transliteration.
  text = ascii_text(text)
Scanning done.
Summary:        info, packages, licenses, copyrights, emails, urls with 1 process(es)
Errors count:   0
Scan Speed:     5.22 files/sec. 14.46 KB/sec.
Initial counts: 5 resource(s): 4 file(s) and 1 directorie(s)
Final counts:   5 resource(s): 4 file(s) and 1 directorie(s) for 11.09 KB
Timings:
  scan_start: 2023-01-30T195301.642569
  scan_end:   2023-01-30T195303.761515
  setup_scan:licenses: 1.29s
  setup: 1.29s
  scan: 0.77s
  total: 2.17s
Removing temporary files...done.

```

For me I had to set this in the main file, and not inside a function.